cli/azd/cmd/infra_create.go

```diff
@@ -272,25 +276,78 @@ func (ica *infraCreateAction) Run(ctx context.Context, cmd *cobra.Command, args
 	return nil
 }
 
-func reportDeploymentStatusInteractive(ctx context.Context, azCli tools.AzCli, env environment.Environment, showProgress func(string)) {
+func reportDeploymentStatusInteractive(ctx context.Context, azCli tools.AzCli, env environment.Environment, spinnerUpdater *spin.SpinnerUpdater, loggedResourcesPtr *map[string]bool) {
```

Comment:
Just wondering - why is `loggedResourcesPtr` typed as a pointer to the map instead of just the map itself?  It seems like we just dereference the value and then never use the pointer part again.  Could this just be `map[string]bool`

Reply:
go handles `map[string]bool` as a pointer.  So, you can remove the `*` from here. You would get a reference to the map and not a copy of the